Lib9c/Action/DPoS/Misc/Asset.cs

```diff
@@ -8,21 +9,24 @@ public struct Asset
             Currency.Legacy("NCG", 2, null);
 
         public static readonly Currency ConsensusToken =
-            Currency.Uncapped("ConsensusToken", 18, minters: null);
+            Currency.Uncapped("ConsensusToken", 0, minters: null);
 
         public static readonly Currency Share =
-            Currency.Uncapped("Share", 18, minters: null);
+            Currency.Uncapped("Share", 0, minters: null);
 
         public static FungibleAssetValue ConsensusFromGovernance(FungibleAssetValue governanceToken)
-        {
-            return new FungibleAssetValue(
-                ConsensusToken, governanceToken.MajorUnit, governanceToken.MinorUnit);
-        }
+            => FungibleAssetValue.FromRawValue(ConsensusToken, governanceToken.RawValue);
+
+        public static FungibleAssetValue ConsensusFromGovernance(BigInteger amount)
+            => ConsensusFromGovernance(GovernanceToken * amount);
+
+        public static FungibleAssetValue ShareFromGovernance(FungibleAssetValue governanceToken)
+            => FungibleAssetValue.FromRawValue(Share, governanceToken.RawValue);
+
+        public static FungibleAssetValue ShareFromGovernance(BigInteger amount)
+            => ShareFromGovernance(GovernanceToken * amount);
```

Comment:
I'm not sure share <-> governance token conversion is really needed, since share is related only to consensus token IMHO.

Reply:
You're right share <-> governace token conversion is not need, but some test needs to be conversion from NCG to share. 
This requires changing from NCG -> power and then power -> share, which is annoying, so I created a method.
This is just for testing purposes, so I will look for alternative methods.

Reply:
https://github.com/planetarium/lib9c/pull/2521/commits/81a7df4007d464a987bd0e46ae6a4b349f72b6da#diff-f5a7ac9f504734a497b61fb3a14316bfeb48e7b624f398e109348105e21e2439
amended